pkgs/development/tools/build-managers/meson/setup-hook.sh

```diff
@@ -25,3 +25,17 @@ if [ -z "$dontUseMesonConfigure" -a -z "$configurePhase" ]; then
     setOutputFlags=
     configurePhase=mesonConfigurePhase
 fi
+
+mesonCheckPhase() {
+    runHook preCheck
+
+    echo "meson check flags: $mesonCheckFlags"
+    meson test $mesonCheckFlags
+
+    runHook postCheck
+}
+
+if [ -z "$dontUseMesonCheck" -a -z "$checkPhase" ]; then
+    setOutputFlags=
```

Comment:
This line does not seem applicable to the check phase.

Reply:
Right, I am sure why I copied it. It does not do anything even in the `configurePhase`.